q2_longitudinal/tests/test_longitudinal.py

```diff
@@ -284,6 +284,28 @@ def test_linear_mixed_effects(self):
             group_categories='delivery,diet,antiexposedall',
             individual_id_column='studyid', metric='observed_otus')
 
+    def test_linear_mixed_effects_no_group_categories(self):
+        linear_mixed_effects(
+            output_dir=self.temp_dir.name, table=None,
+            metadata=self.md_ecam_fp, state_column='month',
+            individual_id_column='studyid', metric='observed_otus')
+
+    def test_linear_mixed_effects_with_random_effects(self):
```

Comment:
You could test for the presence of the new row in the top table of this viz:

```python
self.assertTrue('Random effects</th><td>[month]' in viz)
```

Reply:
I now do a full frame comparison on the model results, which goes above and beyond this suggestion. Thanks!